pycrostates/viz/tests/test_cluster_centers.py

```diff
@@ -78,6 +84,29 @@ def test_plot_cluster_centers():
     with pytest.raises(TypeError, match="'show' must be an "):
         plot_cluster_centers(cluster_centers, info, show=101)
 
+    # gradient
+    plot_cluster_centers(cluster_centers, info, show_gradient=True)
+    plt.close("all")
+
+    # gradient_kwargs
+    plot_cluster_centers(
+        cluster_centers,
+        info,
+        show_gradient=True,
+        gradient_kwargs={"color": "red"},
+    )
+    plt.close("all")
+
+    caplog.clear()
+    plot_cluster_centers(
+        cluster_centers,
+        info,
+        show_gradient=False,
+        gradient_kwargs={"color": "red"},
+    )
+    plt.close("all")
+    assert "`gradient_kwargs` has not effect" in caplog.text
```

Comment:
```suggestion
    assert "argument 'gradient_kwargs' has not effect when" in caplog.text
```

To go with the other suggestion above.